frontend/integration-tests/tests/olm/descriptors.scenario.ts

```diff
@@ -167,6 +167,7 @@ describe('Using OLM descriptor components', () => {
   });
 
   it('displays list containing custom resource', async() => {
+    await crudView.resourceRowsPresent();
```

Comment:
I needed this check to run the tests locally. It appears the loading indicator can disappear briefly while discovery is in flight, which causes the `rowForName` check to run before the content is loaded. I'm not sure why I see in locally, but not in Jenkins.